### DISCUSSION TITLE
Difference between sealos and helm

### DISCUSSION THREAD

**Discussion by fanux** [labring/sealos#1078] [Q&A] [answered]

**Comment by SignorMercurio** [comment, marked as answer]
- helm doesn't care about Kubernetes installation and lifecycle, but sealos does.
- helm doesn't care about how to store and deliver docker images in the chart, but sealos does.
- helm doesn't embed Kubernetes and all the dependencies, but sealos does.
- helm doesn't package binary files, but sealos does.
- sealos has an embeded registry to save all docker images the cluster needs.

Best Practise: Use helm to manage all your Kubernetes manifests, and use sealos to package all the dependencies like docker image and Kubernetes itself. Example: [Building an ingress cluster images](https://github.com/labring/sealos/blob/main/docs/4.0/build-example-ingress-helm.md)
